tests/test-class-amp-theme-support.php

```diff
@@ -212,6 +212,54 @@ public function test_prepare_response() {
 		$this->assertInstanceOf( 'DOMAttr', $removed_nodes['onclick'] );
 	}
 
+	/**
+	 * Test validate_non_amp_theme.
+	 *
+	 * @global WP_Widget_Factory $wp_widget_factory
+	 * @global WP_Scripts $wp_scripts
+	 * @covers AMP_Theme_Support::validate_non_amp_theme()
```

Comment:
There is no such method as `AMP_Theme_Support::validate_non_amp_theme()`. It should be `AMP_Theme_Support:: prepare_response()`.